src/js/init-favorites-buttons.js

```diff
@@ -46,3 +57,7 @@ function showRemoveFavoritesButton() {
   addFavoritesButton.classList.add('hidden');
   removeFavoritesButton.classList.remove('hidden');
 }
+
+function currentPageIsFavorites() {
+  return document.querySelector('body').classList.contains('js-favorites');
```

Comment:
```suggestion
  return document.querySelector('.toggle-btn-favorites').classList.contains('active');
```
Можна обійтись без додавання зайвого класу, у нас вже є перемикач по якому можна визначити чи ми зараз на favorites.

Reply:
На body надійніше, бо скріпт, якій накладає active може не спрацювати.
Крім того - як інший розробник дізнається, що від його класу active , який він вирішив перейменувати, залежить робота кількох функцій ?
Тоді треба на .toggle-btn-favorites крім класу active, який відповідає за візуал, накласти ще клас js-active, який відповідає за скрипти.